src/cascadia/TerminalApp/GlobalAppSettings.cpp

```diff
@@ -213,7 +213,7 @@ void GlobalAppSettings::LayerJson(const Json::Value& json)
 // - <none>
 void GlobalAppSettings::AddColorScheme(ColorScheme scheme)
 {
-    std::wstring name{ scheme.GetName() };
+    std::wstring name{ scheme.Name() };

```

Comment:
_i hate that we're going from wstring (inside scheme) -> hstring -> wstring -> hstring again_

Reply:
_yeaaaaah_. Once every settings object is a winrt type, everything will just be an hstring all the way through.

I should've checked this actually, but does `JsonUtils::GetValueForKey()` work for an hstring?

Reply:
Yes.